server/api-manager.js

```diff
@@ -1020,6 +1050,23 @@ class MindbodyQueries {
     }
     return false;
   }
+
+  loadConfig() {
+    fs.readFile("./server/config/reports.json", "utf8", (err, jsonString) => {
+      if (err) {
+        console.log("File read failed:", err);
```

Comment:
Good stuff. I wonder what happens if the file fails to read, though. I suspect that this.backoffMultiplier and such will be undefined and cause reference errors later. So maybe instead:
        console.log("api-manager config file read failed:", err);
        this.backoffMultiplier = 2;
        this.maxRetries = 5;
        this.initialDelay = 500;

...Actually, on second thought, those defaults probably belong in the constructor. But the same idea.

Reply:
Get this error when trying to run app.js:
Mindbody report server listening at http://localhost:8080
File read failed: [Error: ENOENT: no such file or directory, open '.server/config/reports.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '.server/config/reports.json'

File path shouldn't be ./server/config/reports.json
File path needs to be ./config/reports.json

Reply:
Asher's got a point, but it's up to us where we want to run the thing from. This code works from the repository root, from mindbody-report-generation instead of mindbody-report-generation/server. vscode run it in debugging sessions from that folder for me, so this makes some sense.
I dunno. I'll ask the team on Slack

Reply:
Yeah, that is probably a good improvement to make, I didn't realize imports gave that behavior, Daniel++